packages/paste-core/components/combobox/__tests__/__snapshots__/index.spec.tsx.snap

```diff
@@ -167,7 +167,8 @@ exports[`Combobox  Render should render 1`] = `
   box-shadow: shadowCard;
   list-style-type: none;
   margin: space0;
-  overflow: hidden;
+  max-height: size50;
```

Comment:
Doesnt look like the token was converted to css here and the line above it :O

Reply:
So no changes here?

Reply:
Ah, good idea!
LGTM